source/_integrations/deconz.markdown

```diff
@@ -267,6 +267,41 @@ automation:
     mode: restart
 ```
 
+{% endraw %}
+
+#### Colored Flashing - RGB Philips Hue bulb using deconz.configure
+Note: Requires `on: true` to change color while the bulb is off. If `on:true` is specified, the bulb remains on after flashing is complete. The previous color is not saved.
+
+{% raw %}
```

Comment:
Why is this escaped? Doesn't seem needed?

Reply:
Why is this marked resolved? The template is still escaped without reason. Please remove it.

Reply:
If you're referring to the {% raw %} {% endraw %} tags, I just followed the format of the other published examples.

Reply:
Ok, but it isn't needed here, so please remove those.

Reply:
Ok, removed the tags in the latest commit.